contracts/PublicResolver.sol

```diff
@@ -1,4 +1,5 @@
 pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
```

Comment:
I don't like using experimental, I really don't.

Reply:
Nor do I, but it's the only way to implement this. It's a fairly basic use of it, too, so I think it's safe.